Defs/ThingDefs_Misc/Weapons_Turrets.xml

```diff
@@ -177,6 +180,7 @@
       <ShotSpread>0.1</ShotSpread>
       <SwayFactor>1.42</SwayFactor>
       <RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+      <Mass>50</Mass>
```

Comment:
this could probably be a bit more, would push the smol cart too much tho :shrug: